test/CodeToRest_test.py

```diff
@@ -636,11 +649,617 @@ def test_35(self):
                 ' continuing on this line\n'
                 'and ending on this line \n', ['Powershell'])
 
+    # GAS
+    def test_37_a(self):
+        self.mt('# comment\n',
+                sl(-3) +
+                'comment\n', ['GAS'])
+
+    def test_37_b(self):
+        self.mt('        .text\n'
+                '# comment\n',
+                bf +
+                '         .text\n' +
+                ef +
+                sl(-2) +
+                'comment\n', ['GAS'])
+    """
+    def test_38(self):
+        self.mt('/*\n'
+                'multi-\n'
+                'line\n'
+                'comment\n'
+                '*/\n',
+                sl(-3) +
+                '\nmulti-\n'
+                'line\n'
+                'comment\n\n', ['GAS'])
+    """
+
+    # autohotkey
+    """

```

Comment:
I've fixed this bug in my repo and will push changes soon. Would you un-comment these tests?